could not run rostest-test_label_image_decomposer.xml

```
$ rostest label_image_decomposer.test 
... logging to /home/k-okada/.ros/log/rostest-p40-yoga-3689.log
[ROSUNIT] Outputting test results to /home/k-okada/.ros/test_results/jsk_perception/rostest-test_label_image_decomposer.xml
INFO: cannot create a symlink to latest log directory: [Errno 2] No such file or directory: '/home/k-okada/.ros/log/latest'
/usr/local/lib/python2.7/dist-packages/skimage/util/dtype.py:110: UserWarning: Possible precision loss when converting from float64 to uint8
  "%s to %s" % (dtypeobj_in, dtypeobj))
/usr/local/lib/python2.7/dist-packages/skimage/util/dtype.py:110: UserWarning: Possible precision loss when converting from float64 to uint8
  "%s to %s" % (dtypeobj_in, dtypeobj))
[ERROR] [WallTime: 1475308531.550940] bad callback: <bound method Subscriber.callback of <message_filters.Subscriber object at 0x7f22c9c79290>>
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/topics.py", line 720, in _invoke_callback
    cb(msg)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/message_filters/__init__.py", line 74, in callback
    self.signalMessage(msg)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/message_filters/__init__.py", line 56, in signalMessage
    cb(*(msg + args))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/message_filters/__init__.py", line 199, in add
    self.signalMessage(*msgs)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/message_filters/__init__.py", line 56, in signalMessage
    cb(*(msg + args))
  File "/home/k-okada/catkin_ws/ws_jsk_recognition/src/jsk_recognition/jsk_perception/node_scripts/label_image_decomposer.py", line 130, in _apply
    result_img = fromimage(result_img_pil, mode='RGB')
TypeError: fromimage() got an unexpected keyword argument 'mode'

^Ctesttest_label_image_decomposer ... ERROR!
```

travis ないではpipがつかわれているんだろうか＿
```
Warnings   << jsk_recognition_utils:make /workspace/ros/ws_jsk_recognition/logs/jsk_recognition_utils/build.make.000.log

In file included from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/ndarraytypes.h:1777:0,


                 from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/ndarrayobject.h:18,


                 from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/arrayobject.h:4,


                 from /workspace/ros/ws_jsk_recognition/build/jsk_recognition_utils/python/jsk_recognition_utils/color.c:352:


/usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]


 #warning "Using deprecated NumPy API, disable it by " \


  ^


In file included from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/ndarraytypes.h:1777:0,


                 from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/ndarrayobject.h:18,


                 from /usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/arrayobject.h:4,


                 from /workspace/ros/ws_jsk_recognition/build/jsk_recognition_utils/python/jsk_recognition_utils/nms.c:352:


/usr/local/lib/python2.7/dist-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:15:2: warning: #warning "Using deprecated NumPy API, disable it by " "#defining NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]


 #warning "Using deprecated NumPy API, disable it by " \


  ^
```
rosdistro/rosdep$ grep scipy python.yaml  | grep trusty
    trusty: [python-scipy]
    trusty_python3: [python3-scipy]
```
なので、scipyをpipからインストールするrosdepはない、けど、どこか別のpipプログラムを最初にインストールすると依存関係で、pipのscipyになっているのかな．

--- a/jsk_perception/node_scripts/label_image_decomposer.py
+++ b/jsk_perception/node_scripts/label_image_decomposer.py
@@ -101,7 +101,6 @@ class LabelImageDecomposer(ConnectionBasedTransport):
             # plot label titles on image using matplotlib
             import StringIO
             import PIL.Image
-            from scipy.misc import fromimage
             plt.subplots_adjust(left=0, right=1, top=1, bottom=0,
                                 wspace=0, hspace=0)
             plt.margins(0, 0)
@@ -126,9 +125,7 @@ class LabelImageDecomposer(ConnectionBasedTransport):
             # convert plotted figure to np.ndarray
             f = StringIO.StringIO()
             plt.savefig(f, bbox_inches='tight', pad_inches=0)
-            result_img_pil = PIL.Image.open(f)
-            result_img = fromimage(result_img_pil, mode='RGB')
-            label_viz = result_img
+            label_viz = np.array(PIL.Image.open(f))[:, :, :3]  # RGBA -> RGB
 
         label_viz_msg = bridge.cv2_to_imgmsg(label_viz, encoding='rgb8')
         label_viz_msg.header = img_msg.header

